tests/foreman/ui/test_host.py

```diff
@@ -1377,7 +1377,7 @@ def test_positive_global_registration_end_to_end(
     GlobalParameter().set({'name': 'host_registration_insights', 'value': 1})
     GlobalParameter().set({'name': 'host_registration_remote_execution', 'value': 1})
     # rex interface
-    iface = 'eth0'
+    iface = 'ens3'
```

Comment:
Is this interface applicable to the test.only when using the virtual machine class?

Reply:
That's what I figured from the automation logs. Successful PRT run in 8356 tells me eth0 is ok for the master branch.

Reply:
Perfect, thanks for looking into that